auth_otp/services/kavenegar_service.py

```
@@ -0,0 +1,232 @@
+            }
+            
+            # اگر token2 یا token3 نیاز است
+            # params['token2'] = 'مقدار'
+            # params['token3'] = 'مقدار'
+            
+            response = self.api.verify_lookup(params)
+            
+            # لاگ موفقیت
+            logger.info(
+                f"OTP sent successfully to {phone_number}, "
+                f"message_id: {response['messageid']}"
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)